net-mail/offlineimap/offlineimap-7.3.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Normally you need only one version of this.
+DISTUTILS_SINGLE_IMPL=1
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads,sqlite,ssl?"
+
+inherit distutils-r1
+
+DESCRIPTION="Powerful IMAP/Maildir synchronization and reader support"
+HOMEPAGE="https://www.offlineimap.org/"
+SRC_URI="https://github.com/OfflineIMAP/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc ssl"
```

Comment:
I can't see what USE=ssl does. Is it needed?

Reply:
Since the `PYTHON_REQ_USE` line has `ssl?`, I thought it meant that if OfflineIMAP is built with USE=ssl, Python also has to be built with USE=ssl

Reply:
Looks like that's right :)